site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+Translations for the compatibility ratings and sorting categories for ROMs had been completed for many languages, but never officially implemented in Citra! This PR just fixes this little oversight and gives these ratings in many languages from across the world.
+
+# Surprise Announcement
+
+Last but not least, we are happy to announce that one of our most requested features is now in development!
+
+Developer [GPUCode](https://github.com/GPUCode) has been hard at work adding a brand-new Vulkan backend to Citra! Vulkan is a graphics API similar to OpenGL which is currently used for Citra’s video backend. Unlike OpenGL, however, Vulkan allows for a much lower-level access to the GPU, allowing us to make use of optimizations that are currently not possible in OpenGL.
+
+Please note that this feature is still very much a Work-In-Progress, but initial tests have been very promising already! On some configurations performance in games is more than doubled, with Android devices receiving a significant speedup, especially for those using a Mali GPU.
+Mali GPUs, which are commonly included with Exynos and MediaTek SoCs, are known for having poor OpenGL ES drivers. This causes devices with those GPUs to have low performance on Citra whilst using our OpenGL ES backend. With the addition of Vulkan, Citra is made more accessible to many more people, not just those who have a Snapdragon SoC.
+
+To achieve this feat, [GPUCode](https://github.com/GPUCode) has been undertaking a massive rewrite of the OpenGL backend and the video_core as well, to streamline and simplify the code, making it easier to reuse code between both backends.
+
+Another advantage of Vulkan is being able to use MoltenVK on MacOS devices. We have recently changed our OpenGL requirements from 3.3 to 4.3 (more on this is explained in our [FAQ](https://citra-emu.org/wiki/faq/#system-requirements)) in an effort to modernize Citra's graphical backends. Since Apple has dropped support for modern OpenGL back in 2018, this has resulted in our users on MacOS being stuck on an older version of Citra, not being able to use all the shiny new features we've covered here!
+Using Vulkan, support of Citra will be brought back to thousands of users on their Mac devices!
+
+We are very excited to share more information about Vulkan at a later point, as it is still being worked on every day. Please be patient whilst this massive effort comes to life, as a big project like this does take a long time to materialize!
+
+{{< sidebyside "image" ""
+    "androidOpenGL.png=OpenGL ES on Samsung Galaxy A03, Unisoc T606 (Mali G57). FPS: 9 Speed: 31%"
+    "androidVulkan.png=Vulkan on Samsung Galaxy A03, Unisoc T606 (Mali G57). FPS: 30 Speed: 100%" >}}
+
+# Conclusion
+
+All in all, it has been a tremendous couple of years for Citra development, albeit being a bit slow. We are proud of our developers, new and old, who have shown us the love they have for this project in many ways.
```

Comment:
It might be worth adding a sentence along the lines of:

"We hope this progress report goes a long way to show that Citra development is very much alive and active, despite the silence on our website for a long time. Expect to hear more continued updates from us soon!"